alacritty/src/display.rs

```diff
@@ -394,68 +395,31 @@ impl Display {
             api.clear(background_color);
         });
 
-        let mut lines = RenderLines::new();
         let mut urls = Urls::new();
+        let mut rects: Vec<RenderRect> = Vec::new();
 
         // Draw grid
         {
             let _sampler = self.meter.sampler();
 
             self.renderer.with_api(&config, &size_info, |mut api| {
                 // Iterate over all non-empty cells in the grid
-                for cell in grid_cells {
-                    // Update URL underlines
-                    urls.update(size_info.cols().0, cell);
```

Comment:
Warning related to URLs come from this deleted line.  IIRC work has been done on URL handling, so this needs to be updated.

Unfortunately `text_run`s now map to several cells (through `TextRun::cells`).  I don't know if ligatures can be implemented without modifying URL-related code...